operators/antrea-operator-for-kubernetes/v1.7.0/metadata/annotations.yaml

```diff
@@ -0,0 +1,18 @@
+---
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: antrea-operator-for-kubernetes
+  operators.operatorframework.io.bundle.channels.v1: alpha
```

Comment:
The default channel annotation is missing
`operators.operatorframework.io.bundle.channel.default.v1`